src/services/ApiService.ts

```diff
@@ -0,0 +1,9 @@
+import axios from "axios"
+
+// api client
+const API_BASE_URL_V2 = `${process.env.REACT_APP_BACKEND_URL_V2}`
+export const apiService = axios.create({
+  baseURL: API_BASE_URL_V2,
+  // 100 secs
+  timeout: 100000,
+})
```

Comment:
as mentioned in the write-up, this is because the context adds a layer of indirection that could be avoided (ie, just call directly).

we prefer context to avoid prop drilling but in this case, there's no prop drilling as we can just import and use the service. this also hinders variable name lookup in the ts language server as it's unable to tell the exact location of the service, instead having to point you to the context and then clicking from there @__@